lib/utils/dimensions.dart

```diff
@@ -0,0 +1,6 @@
+class Dimensions {
+  static const double ITEM_HEIGHT_8 = 8.0;
```

Comment:
1. If we want to change dimensions across all the app, we can just change it inside Dimensions 
2. You of course can, but it's better to use padding for padding and height for widgets with height :) 
3. It's easier, but `ITEM_HEIGHT_16 `(for me) makes code more readable 

We can easily find out naming of dimens! If we don't need many dimens (HEIGHT_4, HEIGHT_5, ...) then it can easily be named like S, M, L, XL.